test/controllers/github_profiles_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class GithubProfilesControllerTest < ActionDispatch::IntegrationTest
```

Comment:
I don't know what is this element. So, I assume it's anyway necessary @Kimon-Haars. So I will approve it.